src/test/java/com/cloudant/tests/HttpTest.java

```diff
@@ -0,0 +1,87 @@
+package com.cloudant.tests;
+
+import com.cloudant.client.api.CloudantClient;
+import com.cloudant.client.org.lightcouch.internal.URIBuilder;
+import com.cloudant.http.CookieInterceptor;
+import com.cloudant.http.HttpConnection;
+import com.cloudant.test.main.RequiresCloudant;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+public class HttpTest {
+
+    private CloudantClient account;
```

Comment:
As this is a new test it might be nice to use the JUnit `ExternalResource`. If we eventually refactor all the tests to use it then any changes to how we obtain clients for tests can be made in just one place. I also don't think this needs a new client before each test (although there is only one test at the moment) so might be worth using the `ClassRule` e.g.
```java
@ClassRule
public static CloudantClientResource clientResource = new CloudantClientResource();
private CloudantClient account = clientResource.get();
```
and the `setUp` and `tearDown` methods could then be removed.